Main/StackWalker/StackWalker.cpp

```diff
@@ -85,6 +85,7 @@
 #include <stdlib.h>
 #include <tchar.h>
 #include <windows.h>
+#include <SDKDDKVer.h>

```

Comment:
This will not work in older VS versions… as I said; there is no need to include this header...

Reply:
Ok, I will remove it